Mathlib/Geometry/Manifold/VectorBundle/LocalFrame.lean

```diff
@@ -283,3 +301,334 @@ end
 end IsLocalFrameOn
 
 end IsLocalFrame
+
+namespace Module.Basis
```

Comment:
I'm not absolutely convinced about all these results going into the `Module.Basis` namespace. What do you think about putting them in `Trivialization`.

(Aside: IMHO `Trivialization` itself should be in the `Bundle` namespace but this is a question for another day.)